pkg/operator/ceph/cluster/osd/osd.go

```diff
@@ -542,10 +542,10 @@ func (c *Cluster) resolveNode(nodeName string) *rookalpha.Node {
 	// ensure no invalid dirs are specified
 	var validDirs []rookalpha.Directory
 	for _, dir := range rookNode.Directories {
-		if dir.Path == k8sutil.DataDir || dir.Path == c.dataDirHostPath {
-			logger.Warningf("skipping directory %s that would conflict with the dataDirHostPath", dir.Path)
-			continue
-		}
+		// if dir.Path == k8sutil.DataDir || dir.Path == c.dataDirHostPath {
```

Comment:
I have this commented out (for testing, will remove soon) @travisn. This is what you're talking about yes?

Reply:
oh, this was commented out all along? Yes, this is the place.